### PR TITLE
Ensure add-on behaves properly on all windows.

### PR DESCRIPTION
**Pull request by ericawright**
Load message listeners on every window and framescript on every window (and tab within them).
Thanks to the example [here](https://github.com/mozilla/blurts-addon/blob/master/src/privileged/subscripts/EveryWindow.jsm)

fixes: #41

### CODE REVIEW

File: src/privileged/trackers/api.js [hunk 1]
```diff
@@ -40,33 +30,40 @@ class TrackersEventEmitter extends EventEmitter {
...
+      context.extension.getURL("privileged/trackers/EveryWindow.js"));
     return {
       trackers: {
-        async listenForTrackers() {
-          // Set up frame script and message manager.
-          // Must load framescript in order to get access to docShell.
-          const mm = getMostRecentBrowserWindow().getGroupMessageManager("browsers");
+        async unmount(win) {
+          console.log("UNMOUNT CALLED");
+          // considering the window is being closed, do I need to remove the message listeners?
```
Comment (johannhof) [inline src/privileged/trackers/api.js]: phew, good question, maybe it's a good idea, just to be sure?

**Comment by ericawright** [comment]
done. (Adding  `mm.removeDelayedFrameScript...` in another PR since it requires a bit more movement)

**Comment by ericawright** [comment]
done, will add `mm.removeDelayedFrameScript...` in the next PR

**Comment by ericawright** [comment]
nvm, I added it ^

**Comment by johannhof** [comment]
Great, this is also much cleaner now, thank you!

**Pull request merged**